lib/model.js

```diff
@@ -761,7 +761,7 @@ module.exports = function(registry) {
     // if there is atleast one updateOnly property, then we set
     // createOnlyInstance flag in __create__ to indicate loopback-swagger
     // code to create a separate model instance for create operation only
-    const updateOnlyProps = this.getUpdateOnlyProperties();
+    const updateOnlyProps = this.getUpdateOnlyProperties ? this.getUpdateOnlyProperties() : false;
```

Comment:
This checks only whether `getUpdateOnlyProperties` is truthy, not wether it's a callable function.

I think we should add the proper check right now (`typeof this.getUpdateOnyProperties === 'function'`), so that we don't have to touch this line again in the future.